### PR TITLE
gh-1229 Add tests

### PR DESCRIPTION
**Pull request by DmitryBespalov**
# Setting Up Git LFS for snapshot testing

https://docs.github.com/en/github/managing-large-files/versioning-large-files/about-git-large-file-storage

    brew install git-lfs

### CODE REVIEW

File: MultisigIntegrationTests/UI Snapshot Tests/BalanceTableViewCellTests.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            cell.setDetailText("")
+            cell.setSubDetailText("")
+            cell.setImage(UIImage())
+        }
+        assertSnapshot(matching: vc, as: .image(on: .iPhoneX))
+    }
+
+    func testLongStrings() {
+        let vc = CellTestViewController<BalanceTableViewCell>(estimatedHeight: 60) { cell in
+            cell.setMainText(String(repeating: "ETH repeating ", count: 15))
```
Comment (DmitryBespalov) [inline MultisigIntegrationTests/UI Snapshot Tests/BalanceTableViewCellTests.swift:41]: this needs to be fixed actually, because only amounts are shown

Comment (sche) [reply MultisigIntegrationTests/UI Snapshot Tests/BalanceTableViewCellTests.swift:41]: so what is wrong here?

File: MultisigIntegrationTests/UI Snapshot Tests/BalanceTableViewCellTests.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        let vc = CellTestViewController<BalanceTableViewCell>(estimatedHeight: 60) { cell in
+            cell.setMainText("ETH")
+            cell.setDetailText("0,005")
+            cell.setSubDetailText("155,84 USD")
+            cell.setImage(UIImage(named: "ico-ether")!)
+        }
+        assertSnapshot(matching: vc, as: .image(on: .iPhoneX))
+    }
+
+    func testNotConfigured() {
```
Comment (sche) [inline MultisigIntegrationTests/UI Snapshot Tests/BalanceTableViewCellTests.swift]: do we need to test notConfigured/Empty at all?

**Comment by sche** [comment]
How about a test using BalancesVC with mock data to increase coverage for the whole controller?

File: MultisigIntegrationTests/UI Snapshot Tests/BalanceTableViewCellTests.swift (hunk 2 above)
Comment (DmitryBespalov) [reply MultisigIntegrationTests/UI Snapshot Tests/BalanceTableViewCellTests.swift]: These are testing edge cases. Not configured is used because the XIB had some text, so I fixed it. Empty was used to double-check that it works with empty values.

**Comment by DmitryBespalov** [comment]
@sche I want to start small 😉

**Review by sche — APPROVED** [review]
